docs/misc/deprecated.md

```diff
@@ -12,6 +12,15 @@ parent = "mn_use_docker"
 
 The following list of features are deprecated.
 
+### Docker ps 'before' and 'since' options
+
+**Deprecated In Release: [v1.10.0](https://github.com/docker/docker/releases/tag/v1.10.0)**
+
+**Target For Removal In Release: v1.12**
+
+The `docker ps --before` and `docker ps --since` options are deprecated in favor
```

Comment:
```markdown
The `docker ps --before` and `docker ps --since` options were deprecated. Use `docker 
ps --filter=before=...` and `docker ps --filter=since=...` instead.
```